resources/views/exhibitions/templates/details-2024.blade.php

```diff
@@ -9,7 +9,7 @@
         }
     }
 @endphp
-
+<main>
```

Comment:
@JamieC24 this opening `<main>` tag doesn't seem to have a matching close tag. It's possible I've missed something here, but it might also be a mistake? 

It looks like you made the change. Can you remember if there is a corresponding close tag in another template partial?